boltdesignsystem/bolt
# DS-344: [Remove] Bolt Video Component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-344

## Summary

* Removed all of the examples of Bolt Video Component from the repository
* Updated the `travis.yml` file so Travis will correctly handle the Bolt 5.0.0 release

## How to test

Pull down the branch and test on your local. Run `yarn`, `yarn setup`, `yarn build`, `yarn test:js` to confirm that the build updates are working. 

## Release notes

### Bolt Video
Removed the Bolt Video

### Card Component
Removed a card `media.video` prop to prevent the Bolt Video dependency
